decidim-proposals/lib/decidim/api/proposal_type.rb

```diff
@@ -41,17 +47,42 @@ def url
         Decidim::ResourceLocatorPresenter.new(object).url
       end
 
-      def coordinates
-        [object.latitude, object.longitude]
+      def answered_at
+        return unless object.published_state?
+
+        object.answered_at
+      end
+
+      def answer
+        return unless object.published_state?
+
+        object.answer
       end
 
       def meeting
         object.authors.first if object.official_meeting?
       end
 
+      def cost_report
+        return unless proposal_has_costs? && current_settings.answers_with_costs?
+
+        object.cost_report
+      end
+
+      def execution_period
+        return unless proposal_has_costs? && current_settings.answers_with_costs?
```

Comment:
Should also check the `published_state?` method, if not we will leak this information

Reply:
done in [e1adc69](https://github.com/decidim/decidim/pull/14644/commits/e1adc69ddcf8fbe0a256c8004d9ab222f5a59471)